### ISSUE TITLE
SSR - NextJS Support

### ISSUE THREAD

**Issue by omeraplak** [pankod/refine#1084]

**Comment by omeraplak** [comment]
- [ ] Layout
- [ ] syncWithLocation

**Comment by salihozdemir** [comment]
> Attention
```ts
    // disable syncWithLocation for custom resource tables
    if (resourceFromProp) {
        syncWithLocation = false;
    }
```

These rows of code has been commented in `useTable` for server side rendering.

**Comment by salihozdemir** [comment]
Ssr page resources not allowed `options : { route: "hede" }` property.

**Comment by mhrrmk** [comment]
disabled imports and exports  

`packages/core/src/components/antd/datePicker.tsx`
```
// import "antd/es/date-picker/style/index.css";
```   
<br/>

`packages/core/src/components/antd/calendar.tsx`
```
// import "node_modules/antd/dist/antd.css";
```
<br/>

`packages/core/src/index.tsx`
```
// import "node_modules/antd/dist/antd.css";
// export * from "react-query";
```

**Comment by omeraplak** [comment]
We've released 2.0.10.
